DogPicture/DogPicture/TabBar/TabViewControllerCopy.swift

```diff
@@ -0,0 +1,240 @@
+//
+//  MenuViewController.swift
+//  DogPicture
+//
+//  Created by 김동우 on 2021/06/30.
+//
+
+import UIKit
+
+class TabViewControllerCopy: UITabBarController {
+    let centerButton = UIButton()
+    let cameraButton = UIButton()
+    let libraryButton = UIButton()
+    var centerButtonExpanded: Bool = true
```

Comment:
```suggestion
    var isCenterButtonExpanded: Bool = true
```
요론 물음표 느낌도 괜찮습니다 이해하기가 편해요